tests/admin_widgets/test_autocomplete_widget.py

```diff
@@ -53,7 +53,7 @@ def test_build_attrs(self):
             'class': 'my-class admin-autocomplete',
             'data-ajax--cache': 'true',
             'data-ajax--type': 'GET',
-            'data-ajax--url': '/admin_widgets/band/autocomplete/',
+            'data-ajax--url': '/admin_widgets/band/autocomplete/?to_field=id',
```

Comment:
I'd be great if `to_field` was not included if not necessary.

Reply:
I think it's necessary in this case because the FK field `band` is automatically assigned the `to_field='id'` since no `primary_key` is specified on `Band`.